packages/smart-contracts/src/contracts/ERC20FeeProxy.sol

```diff
@@ -58,4 +52,37 @@ contract ERC20FeeProxy {
       _feeAddress
     );
   }
+
+  /**
+   * @notice Call transferFrom ERC20 function and validates the return data of a ERC20 contract call.
+   * @dev This is necessary because of non-standard ERC20 tokens that don't have a return value.
+   * @return The return value of the ERC20 call, returning true for non-standard tokens
+   */
+  function safeTransferFrom(address _tokenAddress, address _to, uint256 _amount) internal returns (bool result) {
+    
+    // solium-disable-next-line security/no-low-level-calls
+    (result,) = _tokenAddress.call(abi.encodeWithSignature(
```

Comment:
We have to call the transferFrom function this way because we don't know it's signature. For non-standard ERC20 it will break the ERC20 interface we used previously.